source/agora/node/GossipProtocol.d

```diff
@@ -61,6 +66,7 @@ public class GossipProtocol
 
         this.tx_cache[tx_hash] = tx;
         this.network.sendTransaction(tx);
+        this.ledger.receiveTransaction(tx);
```

Comment:
Local storage before network communication

Reply:
Yeah I was wondering about that actually. Could we add it to the style guide?